### PR TITLE
dist: Add linter config

### PR DESCRIPTION
**Pull request by jbergstroem**
This is based on the freebsd template with minor changes since this VM is not run in a FreeBSD Jail.

### CODE REVIEW

File: setup/linter/ansible-playbook.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      with_items: ssh_users
+      tags: user
+
+    - name: General | Create authorized_keys for {{ server_user }}
+      authorized_key: user="{{ server_user }}" key="{{ lookup('file', '/tmp/' + item + '.keys') }}"
+      with_items: ssh_users
+      tags: user
+      
+    - name: General | Keep iojs up to date
+      cron: name="update iojs" minute="0" hour="0" day="*/7" job="pkg update -q && pkg upgrade -yq iojs >/dev/null 2>&1"
```
Comment (rvagg) [inline setup/linter/ansible-playbook.yaml]: how up to date is this kept?

**Comment by rvagg** [comment]
lgtm I guess, I wouldn't have chosen FreeBSD but he who does the work gets to choose I guess!

File: setup/linter/ansible-playbook.yaml (hunk 1 above)
Comment (jbergstroem) [inline setup/linter/ansible-playbook.yaml]: Very active: https://svnweb.freebsd.org/ports/head/www/iojs/Makefile?view=log

**Comment by jbergstroem** [comment]
@rvagg finally you have a reason to spend more time with the devil :smiling_imp: (<- red)

**Comment by rvagg** [comment]
:rage: is probably closer for more than reasons of colour

File: setup/linter/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Note that these hostnames are also used in the ansible-inventory file. The IP addresses will need to be updated each time the servers are reprovisioned.
+
+To set up a host, run:
+
+```text
+$ ansible-playbook -i ../ansible-inventory ansible-playbook.yaml
+```
+
+**Users**: The ansible-vars.yaml file contains a list of users who's GitHub public keys are pulled and placed into
```
Comment (bnoordhuis) [inline setup/linter/README.md]: s/who's/whose/?

Comment (jbergstroem) [inline setup/linter/README.md]: Good catch. I copied this from another vm, looks like most configs has the same typo. I'll remedy them in a new PR.

**Comment by jbergstroem** [comment]
This seems to be working well. I'll merge in a day if there are no objections.

**Comment by rvagg** [comment]
lgtm

**Comment by jbergstroem** [comment]
Merged in 6b3867b99f555f3f1bb0b84c81ea20db2ca343b6. Spread that FreeBSD religion :circus_tent:

**Pull request closed without merging**